examples/custom-server/src/custom_server/app.py

```diff
@@ -22,14 +24,16 @@ def get_greeting(name: str) -> str:
 
 
 mcp_app = mcp.streamable_http_app()
-static = StaticFiles(directory=Path(__file__).parent / "static", html=True)
 
 
 app = FastAPI(
     lifespan=lambda _: mcp.session_manager.run(),
 )
 
-# note the order of mounting here,
-# and don't change it unless you know what you're doing
-app.mount("/api", mcp_app)
```

Comment:
We have some upcoming product features that expect that the MCP server is hosted at /mcp (and this is also the default when using FastMCP), so updating to reflect that